src/cmd/serial.js

```diff
@@ -919,8 +920,22 @@ module.exports = class SerialCommand {
 		return promise.finally(cleanUpFn);
 	}
 
+	/**
+	 * This is a wrapper function created so _serialWifiConfig can return the
+	 * true promise state, but the wrapper can still act like it handled any
+	 * failures gracefully as it acted before testing was attempted.
+	 */
+	serialWifiConfig(...args) {
+		return this._serialWifiConfig.apply(this, args)
```

Comment:
prefer the simpler form 👉 `this._serialWifiConfig(...args)`

Reply:
😳 

I'm looking at this code wondering what was going through my head when I wrote it... why would you bind `this` when calling a function on `this`? Probably refactored it into that state and never simplified.